assets/themes/uk_addresses/uk_addresses.json

```diff
@@ -350,19 +379,32 @@
         {
           "id": "uk_addresses_placename",
           "question": {
-            "en": "What is the place or locality for this address?<div class='subtle'>This is additional information that appears before (or instead of) a street name in the address (e.g. \"The Cross\", \"West Business Park\", \"Castle Mews\")</div>"
+            "en": "What is the place or locality for this address?<div class='subtle'>This is additional information that appears before (or instead of) a street name in the address</div>"
           },
           "render": {
-            "en": "<b>{addr:substreet}</b>"
+            "en": "<div class='subtle'>Place (e.g. Castle Mews, West Business Park)</div><div><b>{addr:substreet}</b></div>"
           },
           "freeform": {
             "key": "addr:substreet",
+            "placeholder": "Place (e.g. The Cross, West Business Park, Castle Mews)",
             "addExtraTags": [
+              "not:addr:parentstreet=",
```

Comment:
Are parentstreets and substreets mutually exclusive? Setting 'substreet' will also remove 'not:addr: **parent** street'. Just asking to be sure...

Reply:
Setting substreet also moves the addr:street tag value to the addr:parentstreet key. So given that we are writing a value into addr:parentstreet we also want to remove not:addr:parentstreet=yes if such a tag exists.